LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) Web3 Community
```

Comment:
Since when was Web3 Community created? If it was in 2020, it would be like the following.
```suggestion
Copyright (c) 2020-2021 Web3 Community
```
Otherwise, if older, then the specific year, OR

```suggestion
Copyright (c) 2021 Web3 Community
```